### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by lagoa89** [zippyui/react-date-picker#49]
Hi, 

First thing, thank you for sharing your work :)

I'd like to ensure full accessibility for a web app I'm working on but the picker doesn't seem to support keyboard navigation out of the box (all tabIndex="1", no highlighting for focus). Am I missing something?

I suspect it also doesn't support the ARIA attributes required for screen readers. Is this something you are planning?

Kind regards,

David

**Comment by radubrehar** [comment]
Hi David and thanks for using this. Please have a look at http://zippyui.com/react-date-picker/
I think it behaves decently, and on the default theme you have an outline on the focused day/cell/month.
Does it work well for your case?
In case you need something even more, have a look at `onRenderDay` and `renderDay` props, since they allow you to customize absolutely everything you want when you render a day cell.

Let us know how it goes.

**Comment by lagoa89** [comment]
Thanks for the quick response. Requested that your website is unblocked by my network admins; will check it out as soon as that is done.

**Comment by radubrehar** [comment]
Any update on this?
